clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -151,6 +151,11 @@
             put(topic2, 1);
         }
     });
+    private MetadataResponse deletedMetadataResponse = RequestTestUtils.metadataUpdateWith(1, new HashMap<String, Integer>() {
```

Comment:
Since this variable only used in the new added test, could we put it into the test?

Reply:
Sure